COPYRIGHT.txt

```diff
@@ -316,6 +316,12 @@ Comment: Minimal PCG32 implementation
 Copyright: 2014, M.E. O'Neill
 License: Apache-2.0
 
+Files: ./thirdparty/misc/polypartition.cpp
```

Comment:
I think we already have this, it's named `triangulator.cpp`.

Reply:
Indeed, it seems to be the same. I'll check if it works with existing version.

Reply:
Existing one seems to be working fine, not sure why it was renamed (and which version is it, README have `Version: TBD, class was renamed`).

Reply:
This was updated in `master` with https://github.com/godotengine/godot/pull/45125 but we didn't backport it to `3.x`.